scripts/get_countries_map_data.R

```diff
@@ -1,34 +1,37 @@
-map_url <- paste0(
-  "http://www.naturalearthdata.com/http//www.naturalearthdata.com/",
-  "download/50m/cultural/ne_50m_admin_0_countries.zip"
-)
+# Note: if the map_url below fails to work, you can also get the dataset
+# manually from the link below:
+# https://hub.arcgis.com/datasets/esri::world-countries-generalized/explore?location=-0.062553%2C0.000000%2C2.00
+
+map_url <-
+  "https://opendata.arcgis.com/api/v3/datasets/2b93b06dc0dc4e809d3c8db5cb96ba69_0/downloads/data?format=shp&spatialRefId=4326&where=1%3D1"
 
-file <- paste0("./scripts/raw_data/", basename(map_url))
+file <- paste0("./scripts/raw_data/", basename(map_url), ".zip")
```

Comment:
Let's use a human-readable name instead of `basename(map_url)`.

Reply:
Done. I've changed this to `world_map_data`.